hooks/000-provide-uids-gids.chroot

```diff
@@ -126,6 +126,7 @@ rtkit:x:115:124:RealtimeKit,,,:/proc:/usr/sbin/nologin
 pipewire:x:116:125:Pipewire,,,:/proc:/usr/sbin/nologin
 polkitd:x:117:126:polkit:/nonexistent:/usr/sbin/nologin
 systemd-journal:x:118:106:Reserved:/run/systemd:/bin/false
+provd:x:119:127:Reserved:/run/provd:/bin/false
```

Comment:
I don't think we need this: provd only adds a group. Not a user.

Reply:
I'll test it, just in case.

Reply:
For what it is worth, we didn't need it in core24-desktop. Provd is started as the gnome-initial-setup user, so I'm not sure how anything could run as this user.

Reply:
removed.